main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        dicts = json.load(file)
+    for name in dicts:
+        p_dict = dicts[name]
+        if Race.objects.filter(name=p_dict["race"]["name"]).exists():
+            race = Race.objects.get(name=p_dict["race"]["name"])
+        else:
+            race = Race.objects.create(name=p_dict["race"]["name"],
+                                       description=p_dict["race"]
+                                       ["description"]
+                                       )
```

Comment:
Відсутупи не читаються
```suggestion
            race = Race.objects.create(
                name=p_dict["race"]["name"], 
                description=p_dict["race"]["description"]
            )
```

Reply:
В інших місцях те саме, попробуй black використовувати, перед комітами

Reply:
Що мені робити коли я використовую black і у мене flake8 помилка іза змін black?

Reply:
Ухти, а що за помилки?